datastore/monitoring/metrics.py

```diff
@@ -149,7 +147,7 @@ def gather_publisher_metrics(
             logger.error(
                 "Exception while creating metrics record for Publisher %s: %s",
                 publisher.name,
-                e,
+                str(e),
```

Comment:
Shouldn't be needed as the `%s` format specifier will call `str()` on `e` and by doing it beforehand it will be run even if logger.error isn't in use.